xsd/netex_part_2/part2_vehicleService/netex_vehicleService_version.xsd

```diff
@@ -230,7 +230,7 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Start time of BLOCK- In principle this can be derived from the Start time of the first journey and the preparation duration but may be stated explicitly as well.</xsd:documentation>
```

Comment:
```suggestion
					<xsd:documentation>Start time of BLOCK. In principle this can be derived from the Start time of the first journey and the preparation duration but may be stated explicitly as well.</xsd:documentation>
```